source/reference/minio-mc/mc-ilm-tier-add.rst

```diff
@@ -235,16 +238,37 @@ The command accepts the following arguments:
 .. mc-cmd:: --storage-class
    :optional:
 
-   The AWS storage class to use for objects transitioned by MinIO. 
-   MinIO supports only the following storage classes:
+   The storage class to apply to objects transitioned by MinIO to the remote bucket.
+   The specified storage class varies depending on the ``TIER_TYPE``.
+   MinIO object transition *requires* the remote storage class to support immediate retrieval (e.g. no rehydration or manual intervention required).
```

Comment:
Maybe use this as the sentence in the other places we talk about this?

Reply:
I'll use the re-worked section, breaking it up I think is clearer.